### PR TITLE
KSQL-351 - Doc error for ksql-server-start

### PR DESCRIPTION
**Pull request by joel-hamill**
https://confluentinc.atlassian.net/browse/KSQL-351

### CODE REVIEW

File: docs/syntax-reference.md [hunk 1]
```diff
@@ -528,10 +528,10 @@ $ cat ksql.properties
 auto.offset.reset=earliest
 
 # Start KSQL in standalone mode with the custom properties above
-$ ksql-cli local --properties-file ./ksql.properties
+$ ksql-cli local ./ksql.properties
```
Comment (hjafarpour) [inline docs/syntax-reference.md]: You will need the --properties-file flag. It would not work without it.

Comment (joel-hamill) [reply docs/syntax-reference.md]: @hjafarpour didn't you say differently here: https://confluentinc.atlassian.net/browse/KSQL-351?focusedCommentId=42416&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-42416

Comment (hjafarpour) [reply docs/syntax-reference.md]: @joel-hamill that was for the ksql-server. The above command is for ksql-cli. The server does not need `--properties-file` but cli needs it.

**Comment by joel-hamill** [comment]
@hjafarpour fixed https://github.com/confluentinc/ksql/pull/440/commits/e863867a3ab4567c6c85b8382757e075b6d761f7

once approved i will merge into https://github.com/confluentinc/docs/pull/549

**Comment by joel-hamill** [comment]
added https://github.com/confluentinc/ksql/commit/24618d14dcd75ef6e299a4fda6cf945b204de3be

**Pull request merged**